webapp/src/Entity/Contest.php

```diff
@@ -564,6 +564,7 @@ public function setStarttimeString(string $starttimeString): Contest
     {
         $this->starttimeString = $starttimeString;
 
+        $this->starttime = $this->getAbsoluteTime($starttimeString);
```

Comment:
If this wasn't here to begin with, I'm wondering if there was maybe a good reason for this.

Reply:
I think in general if we want the same/similar logic for the UI and API, then we should rethink where and how parsing the input is handled in a common place. It feels that this is duplicating code in the import service that's already existing in other places.

Reply:
> If this wasn't here to begin with, I'm wondering if there was maybe a good reason for this.

I had the same question, so I was planning to not merge this PR before having discussed this at the hackathon.


> I think in general if we want the same/similar logic for the UI and API, then we should rethink where and how parsing the input is handled in a common place. It feels that this is duplicating code in the import service that's already existing in other places.

I agree, but than I prefer to move this one for after the release as mentioned in the commit message.